package.json

```diff
@@ -25,11 +25,12 @@
   },
   "homepage": "https://github.com/cronofy/cronofy-node#readme",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "rest": "^1.3.1"
+    "lodash": "^4.17.4",
+    "rest": "^2.0.0"
```

Comment:
Similar to `lodash`, as it worked for `1.3.1` would it be better to specify this as:

```js
"rest": ">=1.0.0 <3.0.0",
```